src/util/hooks.tsx

```diff
@@ -1,8 +1,10 @@
 import { useEffect, useRef, useState } from 'react'
-import { useLocation } from 'react-router-dom'
 import { breakPoint } from './layout'
+import { isBrowser } from './browser'
 
 function getWindowDimensions() {
+  if (!isBrowser()) return { width: 1024, height: 600 }
```

Comment:
Can we use our defined values instead?
```suggestion
  if (!isBrowser()) return { width: layoutWidthMap.lg, height: 600 }
```

Reply:
Great idea